morecantile/data1.0/CanadianNAD83_LCC.json

```diff
@@ -0,0 +1,358 @@
+{
```

Comment:
we don't really have to keep them! They can be found in https://github.com/vincentsarago/tilematrixsets